package-deps-hash/src/getPackageDeps.ts

```diff
@@ -11,63 +11,61 @@ export function getPackageDeps(packagePath: string = process.cwd(), excludedPath
   }
 
   return new Promise((complete) => {
-    child_process.exec(
+    const stdout: string = child_process.execSync(
       `git ls-tree HEAD -r && echo ${PROCESS_OUTPUT_DELIMITER} && git status -s -u .`,
-      { cwd: packagePath },
-      (error: Error, stdout: string) => {
-        const changes: IPackageDeps = {
-          files: {}
-        };
-        const processOutputBlocks: string[] = stdout.split(PROCESS_OUTPUT_DELIMITER);
+      { cwd: packagePath }).toString();
 
-        // Note: The output of git ls-tree uses \n newlines regardless of OS.
-        processOutputBlocks[0].split('\n').forEach(line => {
-          if (line) {
-            const parts: string[] = line.substr(line.indexOf('blob ') + 5).split('\t');
-            if (!excludedHashes[parts[1]]) {
-              changes.files[parts[1]] = parts[0];
-            }
-          }
-        });
+    const changes: IPackageDeps = {
+      files: {}
+    };
+    const processOutputBlocks: string[] = stdout.split(PROCESS_OUTPUT_DELIMITER);
 
-        if (processOutputBlocks[1]) {
-          const filesToHash: string[] = [];
+    // Note: The output of git ls-tree uses \n newlines regardless of OS.
+    processOutputBlocks[0].split('\n').forEach(line => {
+      if (line) {
+        const parts: string[] = line.substr(line.indexOf('blob ') + 5).split('\t');
+        if (!excludedHashes[parts[1]]) {
```

Comment:
This logic would be way more readable if you included a comment showing an example output line, and examples of what the parts[] items look like.